### ISSUE TITLE
【feature】Does the injection rule support taking effect on newly created pods

### ISSUE THREAD

**Issue by nkuacac** [chaosblade-io/chaosblade-operator#253]
## Issue Description

Type: *bug report* or *feature request*

### Describe what happened (or what feature you want)
Inject faults into newly created pods

### How to reproduce it (as minimally and precisely as possible)

1. create inject cr match pod labels
2. new create pod
3.

**Comment by spencercjh** [comment]
No. You will find that any Chaos Engineering open-source project does not support this feature.

**Comment by xcaspar** [comment]
Yes, it's not currently supported. While it's a good suggestion, it's somewhat complex to implement. One solution I can think of is:
1. First, annotate the target pod's annotations with the fault injection information.
2. Monitor pod resource changes, filter the annotation labels, and then perform the fault injection.
We will re-evaluate this feature later.
